lib/msal-common/src/client/BaseClient.ts

```diff
@@ -33,17 +28,11 @@ export abstract class BaseClient {
     protected cryptoUtils: ICrypto;
 
     // Storage Interface
-    protected cacheStorage: ICacheStorage;
+    protected cacheStorage: CacheManager;
```

Comment:
We should either rename CacheManager to CacheStorage, or rename the variables to match the class name.

Reply:
I can rename the variable to cacheManager.